src/Shared/@components/Input/Input.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import s from './Input.module.scss';
+
+export const Input = props => {
+  return (
+    <div className={s.root}>
+      <label htmlFor={props.name}>{props.label}</label>
+      <input type="text" name={props.name} value={props.value} onChange={props.handleChange}></input>
```

Comment:
Notice that you do `**prop**={props.**props**}`. Doesn't look good.

Just pass the props instead, all of them.

```suggestion
      <input {...props}/ >
```

Yes, `onChange` and `handleChange` are different. **WHY THE HELL ARE THEY**? Your custom input should be as close to native in interface as possible. And the handler props is ON CHANGE, it should be called that way. You usually pass handleChange method, yes, so it's `onChange={handleChange}`. What to do **onChange**? Handle it.

Reply:
Yes, senpai!